src/items/item.cpp

```diff
@@ -2286,6 +2301,20 @@ std::string Item::getDescription(const ItemType& it, int32_t lookDistance,
 
 	s << parseImbuementDescription(item);
 
+	if (uint16_t classification = item->getClassification(); classification > 1) {
```

Comment:
You can extract this into a separated function, see example of parseImbuementDescription, and can pass item to reference for avoid nullpointer